litex/tools/litex_json2dts_linux.py

```diff
@@ -44,18 +44,35 @@ def generate_dts(d, initrd_start=None, initrd_size=None, polling=False):
     if initrd_size is None:
         initrd_size = default_initrd_size
 
+    if initrd == "enabled" or initrd is None:
+        initrd_enabled = True
+    elif initrd == "disabled":
+        initrd_enabled = False
+    else:
+        initrd_enabled = True
+        initrd_size = os.path.getsize(initrd)
+
+    if root_device is None:
+        root_device = "ram0"
```

Comment:
I don't think there's any value in providing this - the kernel will try to boot from the initrd as /dev/ram0 if you provide it.